geps/gep-1867.md

```diff
@@ -79,7 +79,7 @@ In order to address the concerns above, I propose a standard `infrastructure` AP
 Note the important part of this is the `Gateway` change; the `GatewayClass` aspect is mostly for consistency.
 
 The exact fields are out of scope for this GEP and will be handled by additional GEPs.
-Some example GEPs already depending on this are [GEP-1713](/geps/gep-1713.md), [GEP-1651](/geps/gep-1651.md), and [GEP-1762](/geps/gep-1762.md)
+Some example GEPs already depending on this are GEP-1713, [GEP-1651](/geps/gep-1651), and GEP-1762
```

Comment:
> Let's just drop the references to GEP-1713 and GEP-1762 for now, and let those GEPs add them back in their own PRs. Looks great otherwise, thanks!!

@kflynn Sorry did you mean drop them entirely? or drop the links?

Reply:
I meant drop the references entirely, sorry! 

> The exact fields are out of scope for this GEP, and will be handled by additional GEPs. One example is [GEP-1651](/geps/gep-1651).

Reply:
Ack done